packages/empty-state/src/BasicEmptyState/BasicEmptyState.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Button from '@leafygreen-ui/button';
+import LeafyGreenProvider from '@leafygreen-ui/leafygreen-provider';
+import { useDarkMode } from '@leafygreen-ui/leafygreen-provider';
+import { isComponentType } from '@leafygreen-ui/lib';
+import { Body, H3, Link } from '@leafygreen-ui/typography';
+
+import {
+  buttonContainerStyles,
+  descriptionStyles,
+  externalLinkStyles,
+  rootStyles,
+  textContainerStyles,
+  titleStyles,
+} from './BasicEmptyState.styles';
+import { BasicEmptyStateProps } from '.';
+
+export function BasicEmptyState({
+  graphic,
+  title,
+  description,
+  PrimaryButton,
+  SecondaryButton,
+  ExternalLink,
+  darkMode: darkModeProp,
+}: BasicEmptyStateProps) {
+  const { theme, darkMode } = useDarkMode(darkModeProp);
+
+  if (!isComponentType(PrimaryButton, 'Button')) {
+    console.error(
+      'The `PrimaryButton` prop in BasicEmptyState should be of type LeafyGreen Button.',
+    );
+  }
+
+  if (!isComponentType(SecondaryButton, 'Button')) {
+    console.error(
+      'The `SecondaryButton` prop in BasicEmptyState should be of type LeafyGreen Button.',
+    );
+  }
+
+  if (!PrimaryButton && !!SecondaryButton) {
+    console.error(
+      'The `SecondaryButton` prop in BasicEmptyState should only be used when the `PrimaryButton` prop is also used.',
+    );
+  }
+
+  return (
+    <LeafyGreenProvider darkMode={darkMode}>
+      <div className={rootStyles}>
+        {!!graphic && <div>{graphic}</div>}
```

Comment:
nit but if we're going to coerce variables to booleans as a pattern lets document this somewhere